### DISCUSSION TITLE
Exporting 3D Coordinates from Avatars in Hubs

### DISCUSSION THREAD

**Discussion by stegoioa** [mozilla/hubs#4175] [General Questions & Discussion] [answered]
Dear all, 

I am working on an exhibition with both a physical and a virtual hubs space. The idea is that live projections in the physical space change color and speed according to the changing positions of users in the Hubs space. The main issue I have with this idea is that I do not know how to export the XYZ coordinates of users live. Is this possible? And if it needs custom coding, any idea where I should start? 

Thanks in Advance!

**Comment by stegoioa** [comment]
thanks

**Comment by matthewbcool** [comment, marked as answer]
You will likely need a custom code solution for what you aim to do. See the hubs-cloud documentation and join our dev community on discord for tips on how to start. https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html
discord invite: https://discord.gg/Jer5t54U

There is a related feature request here: https://github.com/mozilla/hubs/issues/5789

You may also be interested in working within the design constraint to try to produce different results close to what you are going for using various components like uv-scroll and billboarding: https://hubs.mozilla.com/labs/what-is-the-blender-add-on/
